src/metrics/histogram.rs

```diff
@@ -122,6 +122,27 @@ pub fn exponential_buckets(start: f64, factor: f64, length: u16) -> impl Iterato
         .take(length.into())
 }
 
+/// Exponential bucket distribution within a range
+/// /// Creates `length` buckets, where the lowest bucket is `min` and the highest bucket is `max`.
+/// /// The final +Inf bucket is not counted and not included in the returned iterator.
+/// /// The function panics if `length` is 0 or negative, or if `min` is 0 or negative.
```

Comment:
```suggestion
///
/// Creates `length` buckets, where the lowest bucket is `min` and the highest bucket is `max`.
///
/// The function panics if `length` is 0 or negative, or if `min` is 0 or negative.
```

I don't think we should mention INF here, given that we don't do it anywhere else.

Also we don't do double `///` anywhere else.

Reply:
Sure